Worked around [pydata/xarray#2742](https://github.com/pydata/xarray/issues/2742)
by always replacing the default fillvalue for the data type with NaN in arrays
read with Xarray.
